api/src/main/java/io/druid/data/input/impl/LongDimensionSchema.java

```diff
@@ -27,10 +27,16 @@
 {
   @JsonCreator
   public LongDimensionSchema(
-      @JsonProperty("name") String name
+      @JsonProperty("name") String name,
+      @JsonProperty("multiValueHandling") MultiValueHandling multiValueHandling
```

Comment:
multi-value Long columns don't exist, so may as well simplify this and always pass `super(name, null);`